sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_file_client.hpp

```diff
@@ -46,17 +46,6 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
         std::shared_ptr<SharedKeyCredential> credential,
         const ShareClientOptions& options = ShareClientOptions());
 
-    /**
-     * @brief Initialize a new instance of FileClient using token authentication.
-     * @param shareFileUri The URI of the file this client's request targets.
-     * @param credential The client secret credential used to initialize the client.
-     * @param options Optional parameters used to initialize the client.
-     */
-    explicit FileClient(
-        const std::string& shareFileUri,
-        std::shared_ptr<Core::TokenCredential> credential,
```

Comment:
Same here - remove including the `azure/identity/client_secret_credential.hpp` header.